invenio_communities/communities/schema.py

```diff
@@ -130,6 +130,11 @@ class CommunityMetadataSchema(Schema):
     # TODO: Add when general vocabularies are ready
     # domains = fields.List(fields.Str())
 
+    @post_dump
+    def remove_none_values(self, data, **kwargs):
+        """Remove none values."""
+        return {key: value for key, value in data.items() if value is not None}
```

Comment:
Here is the classic discussion between: do we allow the client sending `description: None`, and we strip it out, or we fail and we force the client to send a not null value, or omit the field.

Reply:
Here we are actually cleaning the data that is dumped from our DB, by default marsmallow doesn't allow None values, to do se we need to enable the ` allow_none=True` (meaning that we don't allow None values at the moment)

Reply:
Sorry you are right, it is the dump, not the load :)